packages/view-types/genomic-profiles/package.json

```diff
@@ -23,11 +23,11 @@
     "@vitessce/utils": "workspace:*",
     "@vitessce/vit-s": "workspace:*",
     "d3-array": "^2.4.0",
-    "dynamic-import-polyfill": "^0.1.1",
+    "pixi.js": "^5.0.3",
```

Comment:
why do we need `pixi.js` as a dep?

Reply:
It is a peer dependency of higlass, so I assumed it had to be added here, but I am not sure it is necessary. I will try removing it.

Reply:
It is removed in https://github.com/vitessce/vitessce/pull/1404/commits/048751bef1349fba336ba4d0eff9c5fd3c81023d, everything seems to work without it so I will resolve this